README.md

```diff
@@ -51,6 +51,7 @@ This plugin was tested with:
 -   Docker v20.10.8 (optional):
     -   Since plugin version 0.3.4, Docker is no longer necessary if you opt for a Python environment (more info in [Config](#cairo-version)).
     -   If you opt for the containerized version, make sure you have a running Docker daemon.
+    -   If you use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) install Docker instance on your WSL instead of using Docker Desktop on your windows. Example installation for Ubuntu can be found [here](https://docs.docker.com/engine/install/ubuntu/).
```

Comment:
I would just maybe move this a couple of lines below, below: `On Windows, we recommend using WSL 2.`